client/me/help/main.jsx

```diff
@@ -95,6 +97,11 @@ module.exports = React.createClass( {
 			<Main className="help">
 				<MeSidebarNavigation />
 				<HelpSearch />
+				{ ! user.get().email_verified &&
```

Comment:
As above, we could move this out to a `connect`. Since we're already repeating this, it might even be worth adding a new selector in `/client/state/current-user/selectors.js` to reuse elsewhere:

```js
export function isCurrentUserEmailVerified( state ) {
	const user = getCurrentUser( state );
	if ( ! user ) {
		return false;
	}

	return currentUser.email_verified || false;
}
```